src/storage/exec/IndexOutputNode.h

```diff
@@ -329,9 +334,40 @@ class IndexOutputNode final : public RelNode<T> {
       default: {
         auto v = IndexKeyUtils::getValueFromIndexKey(
             context_->vIdLen(), data.first, col, fields_, context_->isEdge(), hasNullableCol_);
+        if (v.isStr()) {
+          auto iter = std::find_if(fields_.begin(), fields_.end(), [&col](const auto& field) {
+            return col == field.get_name();
+          });
```

Comment:
CHECK whether the iter is valid: `DCHECK(iter != xx.end());`

Reply:
> CHECK whether the iter is valid: `DCHECK(iter != xx.end());`

Just share the discussion details, it is not finished